app/src/test/kotlin/com/wire/android/ui/home/conversations/ConversationsViewModelArrangement.kt

```diff
@@ -33,6 +33,7 @@ import com.wire.kalium.logic.feature.message.MarkMessagesAsNotifiedUseCase
 import com.wire.kalium.logic.feature.message.Result
 import com.wire.kalium.logic.feature.message.SendTextMessageUseCase
 import com.wire.kalium.logic.feature.team.GetSelfTeamUseCase
+import com.wire.kalium.logic.feature.user.IsFileSharingEnabledUseCase
```

Comment:
It would be awesome if we could test this cases, for when sharing is enabled or disabled

Reply:
hmmm sure , I will add some test in kalium for this storage